tests/test_io/test_io_core.py

```
@@ -423,7 +425,226 @@ def test_known_formats_empty_entry_points(self, format_manager):
+
+        manager._eps = pd.Series(
+            {
+                "CYCLE_A__V1": make_loader("CYCLE_B", CycleFormatA),
+                "CYCLE_B__V1": make_loader("CYCLE_A", CycleFormatB),
+            }
+        )
+
+        def load(format_name):
+            try:
+                manager.load_plugins(format_name)
+            except BaseException as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `BaseException`

(BLE001)